apps/portal-api/src/modules/services/deployments/deployments.domain.ts

```diff
@@ -0,0 +1,22 @@
+import { dbUnsecure } from '../../../../knexfile';
+import DeploymentRequest, {
+  DeploymentRequestInitializer,
+  DeploymentRequestMutator,
+} from '../../../model/kanel/public/DeploymentRequest';
+
+export const insertDeploymentRequest = async (
+  deploymentRequest: DeploymentRequestInitializer
+) => {
+  return dbUnsecure<DeploymentRequest>('DeploymentRequest')
+    .insert(deploymentRequest)
+    .returning('*');
+};
```

Comment:
This way you don't have to destructure the returned array in the caller. 
I personnally find it cleaner but it's up to you if you want to do the change 😉 
```suggestion
export const insertDeploymentRequest = async (
  deploymentRequest: DeploymentRequestInitializer
) => {
  const [createdDeploymentRequest] = await dbUnsecure<DeploymentRequest>('DeploymentRequest')
    .insert(deploymentRequest)
    .returning('*');
  return createdDeploymentRequest
};
```